bids_prov/fsl/fsl_config.py

```diff
@@ -15,6 +15,31 @@
 )
 
 
+def get_or_load(fn):
+    """
+    fn should return a json serializable object
```

Comment:
Could you add a few words to state the goal of this function? And what is `fn`?

Reply:
A [comment](https://github.com/Inria-Empenn/BEP028_BIDSprov/blob/boutiques/bids_prov/fsl/fsl_config.py#L20) has been added in the `boutiques` branch.